apps/nextjs-website/src/lib/strapi/types/product.ts

```diff
@@ -0,0 +1,89 @@
+import { StrapiBannerLink } from '@/lib/strapi/types/bannerLink';
+import { Media } from '@/lib/strapi/types/media';
+
+export type BaseProduct = {
+  readonly attributes: {
+    readonly name: string;
+    readonly shortName: string;
+    readonly slug: string;
+  };
+};
+
+export type BaseProductWithBannerLinks = BaseProduct & {
+  readonly attributes: {
+    readonly bannerLinks?: readonly StrapiBannerLink[];
+  };
+};
+
+type StrapiUrl = {
+  readonly id: number;
+  readonly name?: string;
+  readonly url: string;
+  readonly hideTryIt: boolean;
+};
+
+export type ApiDataListPage = {
```

Comment:
You should not export this type, but it will be removed in one of the following PRs
```suggestion
type ApiDataListPage = {
```